packages/shared/src/components/auth/AuthOptions.tsx

```diff
@@ -118,7 +123,6 @@ function AuthOptions({
           name: getNodeValue('traits.name', connected.ui.nodes),
           email: getNodeValue('traits.email', connected.ui.nodes),
           image: getNodeValue('traits.image', connected.ui.nodes),
-          flowId: connected.id,
```

Comment:
I think this was not intended.

Reply:
This should just be the same as the main branch, not sure why there was some change in my commit history